test/controllers/application_controller_test.rb

```diff
@@ -32,7 +33,7 @@ class ApplicationControllerTest < ActionDispatch::IntegrationTest
       get root_path, params: { tab: 'primo' }
       assert_select '#tab-to-target' do
         refute_select '[value=?]', 'primo'
-        assert_select '[value=?]', 'geodata'
+        assert_select '[value=?]', 'all'
```

Comment:
This test is showing an unexpected change. The test says to expect `geodata` but shows `all` is returned.